game_patch/graphics/legacy/gr_d3d.cpp

```diff
@@ -114,23 +114,25 @@ CodeInjection gr_d3d_init_error_patch{
     0x00545CBD,
     [](auto& regs) {
         auto hr = static_cast<HRESULT>(regs.eax);
-        xlog::error("D3D CreateDevice failed (hr 0x{:x} - {})", hr, get_d3d_error_str(hr));
+        xlog::error("D3D CreateDevice failed (hr {})", get_d3d_error_str(hr));
 
-        auto text = std::format("Failed to create Direct3D device object - error 0x{:X} ({}).\n"
+        auto text = std::format("Failed to create Direct3D device object - error {}.\n"
```

Comment:
I have not had a chance to run this code yet, but it seems like two arguments are being provided to std::format (on line 122), yet only one is used after this change?

I'm not understanding why this is being done, it seems like the printed error data is being made less specific?